Library/Homebrew/build_options.rb

```diff
@@ -101,6 +101,15 @@ def unused_options
     @options - @args
   end
 
+  # @private
+  def invalid_options
+    (@options | @args) - @options
```

Comment:
It is to catch imperfect user scenario - most of the time `#invalid_options` will be empty.